### PR TITLE
Fixing some issues with dynamically supporting multiple SSO providers

### PR DESCRIPTION
**Pull request by kevgliss**
@rpicard I had to change a few things around (mostly JS) as this was a bit more difficult to pull off that I thought. If you get a chance I would interested to see how well this would work with a google config.

### CODE REVIEW

File: lemur/static/app/angular/app.js [hunk 1]
```diff
@@ -1,139 +1,158 @@
...
+      'angular-clipboard',
+      'ngFileSaver'
+    ]);
+
+
+  function fetchData() {
+    var initInjector = angular.injector(['ng']);
+    var $http = initInjector.get('$http');
+
+    return $http.get('http://localhost:8000/api/1/auth/providers').then(function(response) {
```
Comment (intgr) [inline lemur/static/app/angular/app.js]: "It worked just fine on the developer's laptop"? :)

Comment (kevgliss) [inline lemur/static/app/angular/app.js]: As part of the gulp `build` step it strips out `http://localhost:8000` leaving `/api/1/auth/providers` which should work fine when deployed. ;-)

Comment (intgr) [inline lemur/static/app/angular/app.js]: Oh right you are, thanks. Turns out my problem was that I ran "gulp build package" in a single command and this way it doesn't do it for some reason.

Comment (kevgliss) [inline lemur/static/app/angular/app.js]: Interesting... it would be nice to fix that if you figure out why that doesn't work.

**Pull request merged**